transitions/add_info_document.js

```diff
@@ -0,0 +1,77 @@
+var infoDocId = function(id) {
+  return id + '-info';
+};
+
+var getSisterInfoDoc = function(db, docId, callback) {
+  db.medic.get(infoDocId(docId), function(err, body) {
+    if (err && err.statusCode !== 404) {
+      callback(err);
+    } else {
+      callback(null, body);
+    }
+  });
+};
+
+var generateInfoDocFromAuditTrail = function(audit, docId, callback) {
```

Comment:
I pulled out the creation of the base info doc. I presume that is what you mean, but I'm not completely clear.